scripts/dendrite_sytest.sh

```diff
@@ -70,7 +70,7 @@ rsync -r --ignore-missing-args --min-size=1B -av /work/server-0 /work/server-1 /
 
 if [ $TEST_STATUS -ne 0 ]; then
     # Build the annotation
-    perl /sytest/scripts/format_tap.pl /logs/results.tap "$BUILDKITE_LABEL" >/logs/annotate.md
+    perl /sytest/scripts/format_tap.pl /logs/results.tap "$GITHUB_SHA" >/logs/annotate.md
```

Comment:
Where does this actually show up? If I understand correctly, this is going from:

```
### TAP output for :rocket: Synapse :: postgres :: workers
```

to 

```
### TAP output for 1b557303bd3e0d8d53a1e1764116e70c266db1bf
```

which seems less useful, but I also don't see any SHA's appearing in the failed or successful output of any of the linked check runs.

Reply:
This puzzled me for a while. I think this is buildkite specific. We used to use https://github.com/matrix-org/annotate-buildkite-plugin to upload annotations.md. I think its functionality is replaced by the `Summarise results.tap` step.

Reply:
Just on BuildKite. I've made it so that we don't run this unless `BUILDKITE_LABEL` is set.